e2e/integration/homePage.js

```diff
@@ -0,0 +1,56 @@
+context( 'Test homepage', () => {
+    before( () => {
+        cy.visit( Cypress.env( 'STAGE_HOST' ) );
+        Cypress.config( 'includeShadowDom', true );
+        cy.get( '#username', { timeout: 5000 } ).type( Cypress.env('USERNAME') );
+        cy.get( '#password' ).type( Cypress.env( 'PASSWORD' ) );
+        cy.get( '#submit' ).click();
+    } );
+
+    it( 'Test for home page header section', () => {
+        cy.get( '.hero__mantra' ).should( 'contain.text', 'Develop Fast·Deliver Faster' );
+        cy.get( '.hero__quote' ).should( 'contain.text', 'One Platform to host your Internal Applications and Services' );
+        cy.get( '.hero__description' ).should( 'contain.text', 'One Portal provides a single place for all internal applications and services, supports consistent User experience by providing standard platform for service hosting and data integration, efficient resource management, real time metrics availability, cross-team collaboration and unified documentation.' );
+        cy.get( '.button--danger' ).should( 'contain.text', 'View Applications' );
+        cy.get( '.button--light' ).should( 'contain.text', 'Built-in Microservices' );
+    } );
+
+    it( 'Test for apps list section', () => {
```

Comment:
Can we name the tests like what exactly it is testing? So for example in this case something like: 'Displays all the apps deployed' to the platform.